README.md

```diff
@@ -183,9 +183,6 @@ Reference sections below in order to know more details about the API
     [(en)](http://telefonicaid.github.io/fiware-orion/api/v2/stable/cookbook)
     [(jp)](https://open-apis.letsfiware.jp/fiware-orion/api/v2/stable/cookbook)
     (Apiary)
-    -   See also NGSIv2 implementation notes
```

Comment:
Now that README.md has been touched, maybe we can take the opportunity to include the link to the new document (just after the bullet about the walkthrough)

Same in the "Reference Documentation" section

(Not sure what to do with the links to the cookbook. Probably they are obsolete, but leave them by the moment.)

Reply:
I think this is better for the last task in order to replace the references to specification or additionally include the reference for the specification.

Reply:
Ok. NTC